### PR TITLE
Update client page info for Spring/Summer 2023 project applications

### PR DESCRIPTION
**Pull request by Lucas-Dunker**
Data-only changes, just simple updates to the site so that clients still applying for the summer aren't confused

image

I was gonna do this yesterday on the bus but I fell asleep instead 😭

### CODE REVIEW

File: src/content/clients/clients.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "reasonsToWorkWithSandbox": [
     "No cost for clients",
     "Work closely with a team of our developers and designers",
-    "A deliverable product in 1-2 semesters"
+    "A deliverable product in 2-3 semesters"
   ],
-  "clientApplicationInfo": "Client applications for Spring 2023 are currently closed. If you’d like to work with us starting in Fall 2023, please fill out our application by clicking the 'Apply now' button below and we’ll be in touch with next steps!",
+  "clientApplicationInfo": "Client applications for Fall 2024 are currently open! If you’d like to work with us starting in Fall 2024, please fill out our project proposal by clicking the 'Apply now' button below and we’ll be in touch with next steps!",
```
Comment (shashjar) [inline src/content/clients/clients.json:7]: Says `Fall 2024` here lol

File: src/content/clients/clients.json [hunk 2]
```diff
@@ -26,8 +26,8 @@
       "buttons": []
     },
     {
-      "header": "Your project has a completion time of 1-2 semesters",
-      "body": "Depending on the project, this timeframe can be extended! However, our most successful projects tend to last 1-2 semesters.",
+      "header": "Your project has a completion time of 2-3 semesters",
+      "body": "Depending on the project, this timeframe can be extended! However, our most successful projects tend to last 2-3 semesters (with a possible summer semester for pass-off and maintenance).",
```
Comment (shashjar) [inline src/content/clients/clients.json:30]: maybe "hand-off" instead of "pass-off"

**Review by shashjar — CHANGES_REQUESTED** [review]
couple of minor changes but otherwise looks good

File: src/content/clients/clients.json (hunk 1 above)
Comment (Lucas-Dunker) [reply src/content/clients/clients.json:7]: lol

**Review by shashjar — APPROVED** [review]
LGTM

**Pull request merged**